src/includes/capture-error/flutter.mdx

```diff
@@ -1,12 +1,19 @@
-In Flutter you can capture any exception object that you caught:
+In Dart you can capture any exception object that you caught:
 
 ```dart
-try {
-  throw null;
-} catch (error, stackTrace) {
-  await sentry.captureException(
-    exception: error,
-    stackTrace: stackTrace,
-  );
-}
+  import 'package:sentry/sentry.dart';
+
+  try {
+    aMethodThatMightFail();
+  } catch (exception, stackTrace) {
+    Sentry.captureException(exception, stackTrace: stackTrace);
```

Comment:
I'm not a big fan of adding `await` everywhere, wdyt @bruno-garcia ? people know it's a future and chose what to do.

Reply:
the pedantic style guide uses `unawaited_futures` https://dart-lang.github.io/linter/lints/unawaited_futures.html

Reply:
those examples are not an `async` method though

Reply:
Currently it's `async` and we submit events async through the client straight to network I/O.

So it needs the `await` in there.

If we decide to use an Isolate to queue submission to the background we could kill the `Future` it returns since it's all in-memory work on the client but it comes with other downsides like if we want to cache to disk (for the underlying layer to submit) we'd need a `hint` to tell the SDK to block (if that's even possible) until I/O completes, say, for the app not to crash before we flush bytes to disk. 

We can discuss going forward the approach but if we don't `await` here we could spawn too many `Future` in a tight loop and cause performance issues if not a crash in the app.

Reply:
let's discuss this, the examples are a pure try/catch method, its not explicit methods are sync or async, its just to show how the API is called, params etc.
I'm fine using `await` everywhere, but it's just noise for me, it does make sense to use `await` when we give examples with `runZonedGuarded` which is an async method by def. and requires `await`